File: src/main/graphql/io/github/graphql/resolver/MarketplaceListingResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for MarketplaceListing
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-05T19:09:00+0800"
)
interface MarketplaceListingResolver {

    @Throws(Exception::class)
    fun logoUrl(marketplaceListingTO: MarketplaceListingTO, size: Int?): String?

}

File: src/main/scala-graphql/io/github/graphql/resolver/MarketplaceListingResolver.scala
package io.github.graphql.resolver

import io.github.graphql.model._

/**
 * Resolver for MarketplaceListing
 */
@javax.annotation.Generated(
    value = Array("com.kobylynskyi.graphql.codegen.GraphQLCodegen"),
    date = "2021-03-06T19:48:01+0800"
)
trait MarketplaceListingResolver {

    @throws[Exception]
    def logoUrl(marketplaceListingTO: MarketplaceListingTO, size: Option[Int]): String

}
